SwiftTweaks/TweakTableCell.swift

```diff
@@ -180,13 +180,11 @@ internal final class TweakTableCell: UITableViewCell {
 			accessory.bounds = textField.bounds
 
 		case .stringList:
-			let textFieldFrame = CGRect(
-				origin: .zero,
-				size: CGSize(
-					width: bounds.width * TweakTableCell.stringTextWidthFraction,
-					height: bounds.height
-				)
-			).integral
+			let textFieldSize = self.textField.sizeThatFits(CGSize(
+				width: CGFloat.greatestFiniteMagnitude,
+				height: bounds.height
+			))
```

Comment:
(Throw in an `.integral` here)

Reply:
Added back. It doesn't show in the final diff now, since it is present already in master, but it's back now on line 194.